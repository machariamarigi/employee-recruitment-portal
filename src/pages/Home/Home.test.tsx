import { shallow } from "enzyme"
import Home from "./Home"

describe("Home testing", () => {
    it("should render Home component", () => {
        const wrapper = shallow(<Home />)
        expect(wrapper).toMatchSnapshot()
    })
})
