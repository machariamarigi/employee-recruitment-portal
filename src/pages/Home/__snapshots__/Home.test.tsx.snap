// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home testing should render Home component 1`] = `ShallowWrapper {}`;
